specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1437,6 +1437,132 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/allowedConnections": {

```

Comment:
Wouldn't it make sense to allow a $filter property to filter the results at this scope?